cypher-shell/src/main/java/org/neo4j/shell/CypherShell.java

```diff
@@ -155,7 +154,7 @@ public boolean isTransactionOpen() {
 
     @Override
     @Nonnull
-    public Optional set(@Nonnull String name, @Nonnull String valueString) throws CommandException {
+    public Optional setParameter(@Nonnull String name, @Nonnull String valueString) throws CommandException {
```

Comment:
I think the raw Optional looks weird?

Reply:
Yes, fixing that in another PR apparently.